_data/simple-icons.json

```
@@ -3533,6 +3533,11 @@
             "hex": "88CE02",
             "source": "https://greensock.com/"
         },
+        {
+            "title": "GridAI",
+            "hex": "78FF96",
+            "source": "https://raw.githubusercontent.com/gridai/logos/main/GridAI-icons/icon-white-48.svg"
```

Before another @simple-icons/maintainers reviews this, it'd be worth updating the source URL to link directly to the file (including blob) that was used.
```suggestion
            "source": "https://github.com/gridai/logos/blob/1e12c83b77abdc22a41566cab232f4db40223895/GridAI-icons/icon-white-48.svg"
```